src/webview/static/base.css

```diff
@@ -338,19 +338,25 @@ body {
   border-radius: 3px;
 }
 
-/* The "subheader" showing sender and timestamp. */
+/* The "subheader" showing sender, emoji status, and timestamp. */
 .subheader {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   margin-bottom: 6px;
 }
-.username {
+.name-and-status-emoji {
+  display: flex;
+  align-items: center;
   font-weight: bold;
   text-overflow: ellipsis;
   white-space: nowrap;
   overflow: hidden;
 }
+.status-emoji {
+  height: 1.4rem;
+  margin-left: 2px;
```

Comment:
Given that it seems to be a bit of a mystery why this is cramped here but not on web, even though web also says `margin-left: 2px` and also seems to not have a space in the HTML, I think a fine strategy would be:
* Go ahead and make this 4px or 6px or whatever seems to make it a reasonable spacing, similar to web.
* Leave a 1- or 2-line comment saying it's different from web, and that's because more is mysteriously needed here in order to get the same effective spacing.

There must be an explanation for the discrepancy somewhere. But it'll be somewhere in the other CSS, and HTML, that's different between the two. Reconciling all that is a bigger job (and we probably never want it *all* to be the same).

Pretty much the worst-case consequence of making it 6px here instead of 2px is that in some future after we change something else to align with web, this spacing will be 4px too much -- so a pretty mild impact. And then if that does happen, a comment here explaining the discrepancy will help us spot the cause and fix it.